src/inference/src/cache_manager.hpp

```diff
@@ -120,15 +120,22 @@ class FileStorageCacheManager final : public ICacheManager {
     void write_cache_entry(const std::string& id, StreamWriter writer) override {
         // Fix the bug caused by pugixml, which may return unexpected results if the locale is different from "C".
         ScopedLocale plocal_C(LC_ALL, "C");
-
+#if defined(_WIN32) && defined(OPENVINO_ENABLE_UNICODE_PATH_SUPPORT)
```

Comment:
Consider add this to define `getBlobFile(id)`, so this function could return correct type. Then here and in `read_cache_entry` it can be just used without `#ifdef`. The `getBlobFile` is private member.